### DISCUSSION TITLE
It's possible to add a countdown?

### DISCUSSION THREAD

**Discussion by rodolphoasb** [timolins/react-hot-toast#188] [Q&A]
I have a `timeLeft` value that comes from the state but I'm having a hard time using dynamic data inside the toast...

Any ideas about how to show a countdown?

**Comment by AzharUddinSheikh** [comment]
> dynamic data inside the toast...

you mean like this? you can use `string literal`

<img width="852" alt="Screenshot 2022-11-16 at 5 39 34 PM" src="https://user-images.githubusercontent.com/73925826/202177183-a1dbadc2-09f0-4c28-9022-c567062e33cc.png">
